libs/sdk-core/src/lib/db/kb/kb.models.ts

```diff
@@ -35,6 +35,7 @@ export interface IKnowledgeBoxCreation {
   description?: string;
   zone: string;
   uuid?: string;
+  allowed_origins?: string[] | null;
```

Comment:
why `null` here?
the question mark make it already possibly `undefined`

Reply:
because we need to pass `null` to reset the field.